apps/aecore/lib/aecore/chain/chain_state.ex

```diff
@@ -91,11 +91,11 @@ defmodule Aecore.Chain.ChainState do
 
   @spec validate_tx(SignedTx.t(), chainstate(), integer()) :: {boolean(), chainstate()}
   defp validate_tx(tx, chainstate, block_height) do
-    try do
```

Comment:
why is the try here removed?

Reply:
That's how the code [has always been](https://github.com/aeternity/elixir-node/blob/c7bee713505ed40e7b1dc2fb4c89fa359704a268/apps/aecore/lib/aecore/chain/block_validation.ex#L119)

Reply:
https://imgur.com/a/RilxQ this is what I see, a catch without a try

Reply:
nevermind, i see what it is now

Reply:
Yes, this is a [implicit](https://github.com/rrrene/credo/issues/224#issue-188811717) version of try/catch